src/AzureExtension/Services/DevBox/DevBoxManagementService.cs

```diff
@@ -36,17 +37,27 @@ public class DevBoxManagementService : IDevBoxManagementService
     public async Task<DevBoxHttpsRequestResult> HttpsRequestToManagementPlane(Uri webUri, IDeveloperId developerId, HttpMethod method, HttpContent? requestContent = null)
     {
         var httpManagementClient = _authService.GetManagementClient(developerId);
-        return await DevBoxHttpRequest(httpManagementClient, webUri, developerId, method, requestContent);
+        var result = await DevBoxHttpRequest(httpManagementClient, webUri, developerId, method, requestContent);
+        return new(JsonDocument.Parse(result.Content).RootElement, new DevBoxOperationResponseHeader(result.ResponseHeaders));

```

Comment:
JsonDocument.Parse can throw an exception.
I am not sure if that is expected by the caller. If caller relies on graceful completion of the async operation, consider to catch exceptions before the "return".